test/functional/apps/visualize/_histogram_request_start.ts

```diff
@@ -24,6 +26,14 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
   describe('histogram agg onSearchRequestStart', function () {
     before(async function () {
+      // loading back default data
+      await esArchiver.load('empty_kibana');
+
+      await kibanaServer.importExport.load('discover');
```

Comment:
This doesn't seem right to load 'discover' saved objects when we're also going to call initTests() which loads 'visualize' saved objects.  But maybe I'm missing something?

Reply:
I believe this was in the previous test's after call, I'll take it out and see.